lib/collector/remote-method.js

```diff
@@ -295,7 +296,7 @@ RemoteMethod.prototype._request = function _request(options) {
     path: options.path,
     headers: this._headers(options),
     agent: agents.keepAliveAgent(),
-    __NR__connection: true // Who measures the metrics measurer?
+    [symbols.connection]: true // Who measures the metrics measurer?
```

Comment:
This is still used to set the `internalOnly` variable, to prevent recording requests (for example, don't instrument http requests to the New Relic collector.)

Reply:
I'll change variable names and comments to make this clearer.

Reply:
i don't see us preventing recording our requests.  We just check for the symbol and then delete it.

Reply:
oh n/m there's logic below it